### DISCUSSION TITLE
[Feature] Immich in file browser

### DISCUSSION THREAD

**Discussion by patienttruth** [immich-app/immich#13577] [Feature Request]
GitHub wont search from my phone so, happy to take a referral to another post if this is a duplicate.

I'm wondering if anyone has discussed adding Immich to the file browser. 

![Screenshot_20241018-060113.png](https://github.com/user-attachments/assets/126c19bc-d824-48e3-ae6f-74f2a27ce6ac)

I was attempting to share some photos taken on another device and realized that since the photos aren't actually on my primary device I'd have to share from the Immich app to the SMS app. Not a big deal, but thought this might be a good opportunity to streamline things, especially if it's just a flag somewhere.

Thanks for all the work put into this awesome app!

**Comment by bo0tzz** [comment]
#1641

**Reply by patienttruth** [reply]
Thank you!
